src/v0/destinations/am/deleteUsers.js

```diff
@@ -27,7 +27,10 @@ const userDeletionHandler = async (userAttributes, config) => {
   };
   // Ref : https://www.docs.developers.amplitude.com/analytics/apis/user-privacy-api/#response
   const batchEvents = getUserIdBatches(userAttributes, DELETE_MAX_BATCH_SIZE);
-  const url = 'https://amplitude.com/api/2/deletions/users';
+  const url =
+    residencyServer === 'EU'
+      ? 'https://analytics.eu.amplitude.com/api/2/deletions/users'
+      : 'https://amplitude.com/api/2/deletions/users';
```

Comment:
Lets use switch case

Reply:
only two cases are there. ESLint suggesting to have at least 3 cases.
ESLint: "switch" statements should have at least 3 "case" clauses(sonarjs/no-small-switch)

Reply:
I see